aten/src/ATen/native/BatchLinearAlgebraKernel.cpp

```diff
@@ -531,6 +531,63 @@ void triangular_solve_kernel(Tensor& A, Tensor& B, Tensor& infos, bool upper, bo
   });
 }
 
+/*
+  Solves the matrix equation A X = B
+  X and B are n-by-nrhs matrices, A is represented using the LU factorization.
+  This is an in-place routine, content of `b` is overwritten.
+
+  Args:
+  * `b` -  [in] the right hand side matrix B
+           [out] the solution matrix X
+  * `lu` - [in] the LU factorization of matrix A (see at::_lu_with_info)
+  * `pivots` - [in] the pivot indices (see at::_lu_with_info)
```

Comment:
nit: pivots returned by `_lu_with_info` encode the sequence of transpositions as written in the doc for `torch.lu`. I have to say the name `pivots` is a bit confusing in general, and lots of people had issues with understanding what `pivots` are until the most recent update of `torch.lu`'s doc.